src/configuration_api.rs

```diff
@@ -0,0 +1,62 @@
+// Built-in uses
+use std::sync::{Arc, RwLock};
```

Comment:
let's also update the README file, and add some 'curl' / 'http' commands that people can copy & run.

Reply:
Sounds good. I'll add a table listing these JSON RPC commands that are added and examples on how to execute them.

Reply:
Added a basic `SUPPORTED_APIS.md` file for now